distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Internal/Replacer.enso

```diff
@@ -95,7 +95,7 @@ build_replacement_vector_cached replacement_string pattern =
    strings and group reference numbers.
 build_replacement_vector : Text -> Regex -> Vector Replacement ! No_Such_Group
 build_replacement_vector replacement_string pattern =
-    replacement_pattern = Regex.compile group_reference_regex
+    replacement_pattern = group_reference_regex
```

Comment:
I'm wondering if this change changes anything?

The module-level definitions are always a 'function', meaning that `group_reference_regex` unfortunately is not a constant that is computed once, but is a 0-argument function that is executed every time the method is called. So we are still calling `Regex.compile` every time.

I still think this change is OK to keep and I prefer to have the `compile` call as part of the definition - if only for the reason that _if_ at some point we get an optimization that will make such constants computed once, this code will benefit from that. But I wanted to point out that in the current state, there is no practical change here (as far as I understand).

Reply:
Yes, I just changed it since nothing needed to use the raw string directly.

Reply:
Yeah, I do agree that having this typed to `Regex` is nicer than a `Text` constant 👍 Just wanted to share the context about such module "constants" as it's not obvious.